common/include/common/common_utils.hpp

```diff
@@ -597,12 +558,10 @@ get_trsm_batched_params(Args& args) {
       for (char uplo : {'u', 'l'}) {
         for (char trans : {'n', 't'}) {
           for (char diag : {'u', 'n'}) {
-            for (index_t m = dmin; m <= dmax; m *= 2) {
-              for (index_t n = dmin; n <= dmax; n *= 2) {
-                trsm_batched_default.push_back(
-                    std::make_tuple(side, uplo, trans, diag, m, n, alpha,
-                                    batch_size, stride_a_mul, stride_b_mul));
-              }
+            for (index_t n = dmin; n <= dmax; n *= 2) {
+              trsm_batched_default.push_back(
```

Comment:
I don't think this `trsm_batched` is going to be used anywhere, since this operator is not available in SYCL-BLAS at the moment. Could you please double check this?

Reply:
Disregard this comment please. I forgot that we use `trsm_batched` for `cublas/rocblas` benchmarks.